internal/integration/api/common.go

```diff
@@ -157,6 +160,105 @@ func (suite *CommonSuite) TestDNSResolver() {
 	}
 }
 
+// TestBaseOCISpec verifies that the base OCI spec can be modified.
+func (suite *CommonSuite) TestBaseOCISpec() {
+	if suite.Cluster != nil && suite.Cluster.Provisioner() == "docker" {
+		suite.T().Skip("skipping ulimits test since provisioner is docker")
+	}
+
+	if testing.Short() {
+		suite.T().Skip("skipping test in short mode.")
+	}
+
+	node := suite.RandomDiscoveredNodeInternalIP(machine.TypeWorker)
+
+	k8sNode, err := suite.GetK8sNodeByInternalIP(suite.ctx, node)
+	suite.Require().NoError(err)
+
+	nodeName := k8sNode.Name
+
+	suite.T().Logf("adjusting base OCI specs on %s/%s", node, nodeName)
+
+	suite.AssertRebooted(
```

Comment:
can't we just restart containerd rather than a reboot?

Reply:
Right now - no, we don't have anything that controls CRI

Reply:
we can fix for 1.10